dom0/sd-gpg.sls

```diff
@@ -17,3 +17,8 @@ sd-gpg:
       - label: purple
     - prefs:
       - netvm: ""
+
+# Add tag for default qubes-rpc deny catch-all rule
+# (see dom0/sd-dom0-qvm-rpc.sls)
+qvm-tags sd-gpg add sd-workstation:
+  cmd.run
```

Comment:
Convert the `cmd.run` calls to `qvm-tags` to a `- tags:  ` block on the VM spec; see the `- prefs:` block as an example. Docs here: https://github.com/QubesOS/qubes-mgmt-salt-dom0-qvm/blob/master/README.rst#qvm-vm Note that the syntax is slightly different when using the `qvm` wrapper lib; e.g.

```
  - tags:
      - add:
        - sd-workstation
```